packages/aws-client/src/client.ts

```diff
@@ -50,13 +45,7 @@ const awsClient = <T extends AWSClientTypes>(
     ...serviceOptions,
   };
   if (version) options.apiVersion = version;
-  if (inTestMode()) {
```

Comment:
Presumably this was a V2 issue that V3 corrects?

Reply:
Correct. Looks like that `serviceIdentifier` issue was a v2 bug.